src/components/FlatList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactNode } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { IMovie } from '@/interfaces/interfaces';
+import { IMAGE_BASE_URL } from '@/utils/constants';
+const FlatList = ({
+  movieLists,
+  round,
+}: {
+  movieLists: IMovie[];
+  round: boolean;
+}) => {
+  return (
+    <div className="container">
+      {movieLists.map((movie, idx) => (
+        <FlatListItem item={movie} key={idx} round={round} />
+      ))}
+      <style jsx>{`
+        .container {
+          display: flex;
+          gap: 0.5rem;
+          width: 40rem;
```

Comment:
40rem을 주신 이유가 있을까요 ?? 이렇게되면 화면이 전체 container 크기인 375px을 넘게될 수 있어서 100%로 부모 크기 맞춰가는 게 좋아 보입니다!
```suggestion
          width : 100%;
```